src/Analyzers/Core/Analyzers/DiagnosticCustomTags.cs

```diff
@@ -16,9 +15,12 @@ internal static class DiagnosticCustomTags
         /// <summary>
         /// it is string[] because DiagnosticDescriptor expects string[]. 
         /// </summary>
```

Comment:
❕ This comment is now in the wrong location (would not be opposed to deleting it)

```suggestion
```

Reply:
Doc comment seems weird, I'll just remove it.